src/js/tracing/timetodisplaynative.tsx

```diff
@@ -27,9 +28,9 @@ let RNSentryOnDrawReporter: HostComponent<RNSentryOnDrawReporterProps> | typeof
  * Native component that reports the on draw timestamp.
  */
 export const getRNSentryOnDrawReporter = (): typeof RNSentryOnDrawReporter => {
-  if (!RNSentryOnDrawReporter) {
+  if (!RNSentryOnDrawReporter && ReactNativeLibraries.ReactNative.requireNativeComponent) {
```

Comment:
`ReactNativeLibraries.ReactNative.requireNativeComponent` I would expect this to throw `Can't read requireNativeComponent of undefined` on the web, as the `ReactNativeLibraries={}` on the web.

Reply:
I moved `ReactNativeLibraries.ReactNative?.requireNativeComponent` below for making the compiler happy on making sure requireNativeComponent is defined  in case of having a  native component.